lib/integrations/awsoidc/deployservice.go

```diff
@@ -425,7 +432,12 @@ func DeployService(ctx context.Context, clt DeployServiceClient, req DeployServi
 
 // upsertTask ensures a TaskDefinition with TaskName exists
 func upsertTask(ctx context.Context, clt DeployServiceClient, req DeployServiceRequest, configB64 string) (*ecsTypes.TaskDefinition, error) {
-	taskAgentContainerImage := fmt.Sprintf(teleportContainerImageFmt, req.TeleportVersionTag)
+	teleportFlavor := teleportOSS
+	if modules.GetModules().BuildType() == modules.BuildEnterprise {
+		teleportFlavor = teleportEnt
+	}
+
+	taskAgentContainerImage := fmt.Sprintf(teleportContainerImageFmt, teleportFlavor, req.TeleportVersionTag)
```

Comment:
> because now I have to scroll to another part of the code

Don't you use "go to definition" and "jump back"? :sweat_smile: 
Looks better, just changed it :+1: